src/core/libraries/np_trophy/np_trophy.cpp

```diff
@@ -223,6 +223,14 @@ int PS4_SYSV_ABI sceNpTrophyGetGameIcon(OrbisNpTrophyContext context, OrbisNpTro
     return ORBIS_OK;
 }
 
+struct GameTrophyInfo {
+    uint32_t numGroups;
```

Comment:
```suggestion
    u32 num_groups;
```
Ditto for all other cases of these changes